start/compute.md

```diff
@@ -548,3 +548,20 @@ The jclouds API allows many `Statements` to be built entirely from high-level co
 without having to resort to OS-specific scripts. This enables developers to express what they mean without having
  to deal with the gory details of various OS flavors.
 To see the commands that will be executed, print the result of `Statement.render(OsFamily.UNIX)`, for example.
+
+### Sharing the credential store between ComputeService instances
+
+By default there's a separate credential store per ComputeService instance. To change that and share the
+same credential store between instances create your ComputeService as follows:
+
+{% highlight java %}
+public static final Map<String, ByteSource> SHARED_CREDENTIAL_STORE = new ConcurrentHashMap<String, ByteSource>();
+...
+Module sharedCredStore = new CredentialStoreModule(SHARED_CREDENTIAL_STORE);
```

Comment:
[minor] Just to clarify: is there any specific reason to pull the `SHARED_CREDENTIAL_STORE` out as a constant, or could one e.g. also have the module as a constant?

I'm asking to avoid users e.g. inlining the declaration if they feel it's not necessary, and ending up with unexpected behaviour.